websites/R/Raven Code/dist/metadata.json

```diff
@@ -9,15 +9,11 @@
 		"en": "Raven Code is an extraordinary community that aims to thrive. Our Discord server has nice channels where you can chat with someone. This server is a warm place where you can meet good friends.",
 		"tr": "Raven Code gelişmeyi hedefleyen olağanüstü bir topluluktur. Discord sunucumuzda birileriyle sohbet edebileceğin güzel kanallar var. Bu sunucu, iyi arkadaşlıklar kurabileceğin sıcak bir ortamdır."
 	},
-	"url": "ravencode.live",
-	"version": "2.0.2",
+	"url": ["ravencode.live", "docs.ravencode.live"],
+	"version": "2.0.3",
 	"logo": "https://i.imgur.com/74Calja.png",
 	"thumbnail": "https://i.imgur.com/LabRRNC.png",
-	"color": "#121316",
+	"color": "#7a92f1",
 	"category": "other",
-	"tags": [
-		"discord",
-		"kod-paylaşım",
-		"raven-code"
-	]
+	"tags": ["discord", "kod-paylaşım", "raven-code"]
```

Comment:
I believe the former version was preferable hence why we had it - revert this change

Reply:
I did not understand

Reply:
Rever the change on this line

Reply:
I didn't do it. Prettier did it.

Reply:
Run `yarn format` or `yarn ms` to reformat this file or revert the change manually. Prettier does mess with the file if you are not careful, and it may be a good idea to only prettify your ts files or undo changes prettier does to the metadata files after running prettier.